### PR TITLE
Option to use double quotes (") as escape character

### PR DESCRIPTION
**Pull request by yegortokmakov**
Our team is using double quotes for strings and we have to manually edit generated interfaces to pass our linter. This PR adds `Settings.doublequoteEscape` which will use double quotes in generated interfaces if set to `true`. Default behavior stays unchanged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,6 +235,11 @@ export interface Settings {
    * @default '  ' (two spaces)
    */
   indentationChacters: string;
+  /**
+   * If set to true, will use double quotes for strings
+   * @default false
+   */
+  readonly doublequoteEscape: boolean;
```
[the remark this answers is not in the document]
Comment (yegortokmakov) [reply README.md]: > Looks great can you remove the word readonly here? I'll get this released for you

done! thanks!

**Review by mrjono1 — APPROVED** [review]

**Comment by mrjono1** [comment]
change released https://github.com/mrjono1/joi-to-typescript/releases/tag/4.14.0

**Comment by mrjono1** [comment]
Thankyou for the contribution

**Comment by yegortokmakov** [comment]
thanks @mrjono1 !

**Pull request merged**